RFC-003-set-up-google-account.md

```diff
@@ -0,0 +1,169 @@
+- We can keep a better historical record of non-GitHub assets like drafts or files in Google Docs, slide decks, etc. The project itself doesn't necessarily need to own these assets as long as they are shared with the project's Google account so that they show up in the shared drive.
+- We could possibly use the mail account for official communications.
+- We could use this official account to create accounts for other free services (such as social media).
+- There's probably even more benefits that I haven't thought to list here!
+
+
+### Regulating account access
+
+Access will be shared with community members on an as-needed basis.
+The goal is to avoid being too restrictive or too liberal with access to this account.
+We want to avoid a situation in which only one person has access to this account because that puts too much control in the hands of one community member.
+If that community member becomes unresponsive for various reasons, it could be difficult to access the account.
+However, we also want to avoid a situation in which most community members have access to the account because that could compromise our project's overall security.
+It might also be advisable to ensure that account access is distributed across a few individuals in a few different time zone regions (such as the US, APEC, and EMEA).
+
+For that reason, when the account is first created, the account credentials will be shared with:
+
+- The current project chair (Alyssa)
+- The vice-chair (we need to create this position and nominate someone to it)
+- The templates coordinator (we need to create this position and nominate someone to it; a future RFC that I'm submitting will make a recommendation for this role and it will likely be Aidan Doherty)
+
+Other individuals may possibly request access to the Google account for these reasons:
+
+- You are leading multiple working groups and need to update meeting details regularly
+- Your working group is creating a lot of files and assets that should be stored in the shared project Google Drive
+- You have been authorized by the project steering committee to create an account with another service using our Google account
+- You need to use the project mail account regularly to communicate with others in an official capacity
+- Additional important reasons that we haven't yet thought of
+
+Members who are granted Google account access should be trusted by the community.
+They should have belonged to and/or contributed to the community for at least a quarter (3 months).
+Being a member of the project steering committee helps but should not necessarily be required.
+
+Access can be granted by reaching out to the project chair or vice-chair, who will raise a motion and encourage voting from the project steering committee members.
+
+
+### Rotating passwords
+
+I propose we set up a cadence for changing our account passwords.

```

Comment:
There also seems to be the ability to [delegate access to the account](https://gsuitetips.com/tips/gmail/delegate-access-to-your-gmail-account/), which might alleviate some of the fears regarding bad actors, and will allow us to easily add/remove members while also relieving users of the need to remember Yet Another Password. I'm not sure if this is the best approach for our purposes, but wanted to at least bring it up.

Reply:
Incorporated this comment into the RFC. Resolving.